# Require a unique command path for each command file listed for the project (`shell-shock/duplicate-command-path`)

ğŸ’¼ğŸš« This rule is enabled in the ğŸ”’ `strict` config. This rule is _disabled_ in
the following configs: ğŸ“‹ `base`, ğŸŒŸ `recommended`.

ğŸ”§ This rule is automatically fixable by the
[`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

## Rule Details

ğŸ‘ Examples of **incorrect** code for this rule:

```ts
const url = new URL("https://example.com");
```

ğŸ‘ Examples of **correct** code for this rule:

```ts
const url = new StormURL("https://example.com");
```

## Version

