# All command files must include a handler function as the default export (`shell-shock/invalid-command-export`)

ğŸ’¼ğŸš« This rule is enabled in the ğŸ”’ `strict` config. This rule is _disabled_ in
the following configs: ğŸ“‹ `base`, ğŸŒŸ `recommended`.

ğŸ”§ This rule is automatically fixable by the
[`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

## Rule Details

ğŸ‘ Examples of **incorrect** code for this rule:

```ts
export default const value = 1;
```

ğŸ‘ Examples of **correct** code for this rule:

```ts
export default function handler(options) {
    ...
}
```

## Version
